deltachat-ffi/deltachat.h

```diff
@@ -6080,12 +6086,29 @@ void dc_event_unref(dc_event_t* event);
 #define DC_EVENT_INCOMING_REACTION        2002
 
 
+
+/**
+ * A webxdc wants an info message or a changed summary to be notified.
+ *
+ * @param data1 contact_id ID of the contact sending.
+ * @param data2 (int) msg_id + (char*) text_to_notify.
+ *      msg_id in dc_event_get_data2_int(), referring to webxdc-info-message
+ *      or webxdc-instance in case of summary change.
+ *      text_to_notify in dc_event_get_data2_str().
+ *      string must be passed to dc_str_unref() afterwards.
```

Comment:
That's different to `DC_EVENT_INCOMING_MSG`, which only contains a msg_id and then the UI calls `getSummarytext()` on the message to get the text that should be shown (which doesn't need to be the same text as is shown in the message). If there is a reason for this, it's fine, we should document it more explicitly in this case.

Reply:
as discussed offline just now, the reason is that we may notify about changes in `update.summary`